.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Motivation and Context
```

Comment:
Just wondering if it's worth adding a checklist (like in some of our other repos) to cue/remind the PR person to update `Context Index` if needed?